src/include/sof/lib/alloc.h

```diff
@@ -64,6 +64,10 @@ struct block_hdr {
 	uint16_t used;		/* usage flags for page */
 } __packed;
```

Comment:
The commit message is empty? Please give more information about why this change is needed. Also please use imperative mood as pointed out in a previous comment.

So, please use something like this:

memory: Introduce API for requesting align buffers

This is needed because....

I am pretty sure everything is clear for you because you worked on that. But please have mercy on some poor
guys wanting to understand the code on months later.